addons/l10n_latam_check/views/account_payment_view.xml

```diff
@@ -0,0 +1,214 @@
+        <field name="priority">20</field>
+        <field name="mode">primary</field>
+        <field name="inherit_id" ref="account.view_account_payment_search"/>
+        <field name="arch" type="xml">
+            <!-- TODO: maybe this filter could go on directly on account.view_account_payment_search -->
+            <filter name="matched" position="after">
+                <filter string="Bank Unmatched" name="unmatched" domain="[('is_matched', '=', False)]"/>
+            </filter>
+
+            <field name="journal_id" position="after">
+                <field name="l10n_latam_check_current_journal_id"/>
+            </field>
+            <filter name="date" position="after">
+                <separator/>
+                <filter string="Checks on hand" name="checks_on_hand"
+                        domain="[('state', '=', 'posted'),
+                                 ('l10n_latam_check_current_journal_id.inbound_payment_method_line_ids.payment_method_id.code', '=', 'in_third_checks')]"/>
+                <!-- TODO: not possible with selected_payment_method and the ilike?-->
+            </filter>
+            <filter name="journal" position="after">
+                <filter name="groupby_third_check_current_journal" string="Third Check Current Journal" domain="[]"
+                        context="{'group_by': 'l10n_latam_check_current_journal_id'}"/>
+            </filter>
+            <filter name="groupby_date" position="after">
+                <filter string="Check Payment Date" name="groupby_l10n_latam_check_payment_date" domain="[]" context="{'group_by': 'l10n_latam_check_payment_date'}"/>
+            </filter>
+        </field>
+    </record>
+
+    <record id="view_account_payment_form_inherited" model="ir.ui.view">
+        <field name="name">account.payment.form.inherited</field>
+        <field name="model">account.payment</field>
+        <field name="inherit_id" ref="account.view_account_payment_form" />
+        <field name="arch" type="xml">
+            <sheet position="before">
+                <div class="alert alert-warning mb-0" role="alert" attrs="{'invisible': [('l10n_latam_check_warning_msg', '=', False)]}">
+                    <field name="l10n_latam_check_warning_msg" nolabel="1"/>
+                </div>
+            </sheet>
+            <field name="check_number" position="attributes">
+                <attribute name="attrs">{'invisible': ['|', ('payment_method_code', '!=', 'check_printing'), ('check_number', '=', False), ('l10n_latam_checkbook_id', '=', False)], 'readonly': ['|', ('state', '!=', 'draft'), ('l10n_latam_checkbook_id', '=', False)], 'required': [('l10n_latam_checkbook_id', '!=', False)]}</attribute>
+            </field>
+
+            <xpath expr="//div[@name='amount_div']//field[@name='amount']" position="attributes">
+                <attribute name="attrs">{'readonly': ['|', ('state', '!=', 'draft'), ('payment_method_code', 'in', ['in_third_checks', 'out_third_checks'])]}</attribute>
```

Comment:
Sorry, I don't understand

Reply:
ahh, now I get it.
Allow to modify but raise on post?  for me no problem, maybe less code but ugglier on usability
@vbe-odoo

Reply:
So if I understand correctly, if we remove this, the user will be able to edit the amount of the third check and only see the block when posting? 
If that's the case, it will be annoying the 1st time, but also, they shouldn't change that so it will be weird if they try to do that.
cc' @jjscarafia @jco-odoo

Reply:
done, removed